### PR TITLE
added checks for friends

### PR DESCRIPTION
**Pull request by SwiftHD**
Hope its all good this time!

### CODE REVIEW

File: Horion/Module/Modules/TriggerBot.cpp [hunk 1]
```diff
@@ -16,20 +17,37 @@ const char* TriggerBot::getModuleName()
...
 	Odelay++;
 	if (target != 0 && Odelay >= delay)
 	{
+		if (target == g_Data.getLocalPlayer()) // Skip Local player
+			return;
+
+		if (target == 0)
+			return;
+
+		if (target->timeSinceDeath > 0 || target->damageTime >= 7)
```
Comment (richardletshacks) [inline Horion/Module/Modules/TriggerBot.cpp:33]: What has this to do with friends

Comment (SwiftHD) [reply Horion/Module/Modules/TriggerBot.cpp:33]: friends and valid entities.... trigger bot used to just smack anything

Comment (SwiftHD) [reply Horion/Module/Modules/TriggerBot.cpp:33]: it used to hit npc and shit in servers now it only hits players that are not your friend

Comment (richardletshacks) [reply Horion/Module/Modules/TriggerBot.cpp:33]: Add AntiBot module and build in check to Aura and Triggerbot? Would have more use

File: Horion.vcxproj [hunk 2]
```diff
@@ -22,7 +22,7 @@
     <VCProjectVersion>15.0</VCProjectVersion>
     <ProjectGuid>{499784D2-7EA3-4CBE-BD93-EC0DBBC42FCD}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
-    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.18362.0</WindowsTargetPlatformVersion>
```
Comment (ghost) [inline Horion.vcxproj]: um no

File: Horion/Module/Modules/TriggerBot.cpp (hunk 1 above)
Comment (SwiftHD) [reply Horion/Module/Modules/TriggerBot.cpp:33]: someone else can do that. I just edited trigger bot on request from a guy in discord.

File: Horion/Module/Modules/TriggerBot.cpp [hunk 3]
```diff
@@ -16,20 +17,36 @@ const char* TriggerBot::getModuleName()
...
+			return;
+
+		if (FriendList::findPlayer(target->getNameTag()->getText())) // Skip Friend
+			return;
+
+		if (!Target::isValidTarget(target))
+			return;
+
+		else {
```
Comment (ghost) [inline Horion/Module/Modules/TriggerBot.cpp:42]: else is not needed

**Review by ghost — CHANGES_REQUESTED** [review]

File: Horion.vcxproj (hunk 2 above)
Comment (SwiftHD) [reply Horion.vcxproj]: thats on my end. its weird i cant unselect it and building is broken on 10.0.17763.0 for me

**Comment by SwiftHD** [comment]
idk how to delete so im just closing

**Pull request closed without merging**